src/controllers/shared/base.resume.submission.controller.ts

```diff
@@ -0,0 +1,107 @@
+import { NextFunction, Request, Response } from "express";
+import { v4 as uuidv4 } from "uuid";
+import { Session } from "@companieshouse/node-session-handler";
+
+import * as config from "../../config";
+
+import { ApplicationData } from "../../model";
+import { createAndLogErrorRequest, logger } from "../../utils/logger";
+import { setExtraData } from "../../utils/application.data";
+import { isActiveFeature } from "../../utils/feature.flag";
+import { getOverseasEntity } from "../../service/overseas.entities.service";
+import { HasSoldLandKey, ID, IsSecureRegisterKey, OverseasEntityKey, Transactionkey } from "../../model/data.types.model";
+import { WhoIsRegisteringKey, WhoIsRegisteringType } from "../../model/who.is.making.filing.model";
+import { OverseasEntityDueDiligence, OverseasEntityDueDiligenceKey } from "../../model/overseas.entity.due.diligence.model";
+import { DueDiligence, DueDiligenceKey } from "../../model/due.diligence.model";
+import { BeneficialOwnerGov, BeneficialOwnerGovKey } from "../../model/beneficial.owner.gov.model";
+import { BeneficialOwnerIndividual, BeneficialOwnerIndividualKey } from "../../model/beneficial.owner.individual.model";
+import { BeneficialOwnerOther, BeneficialOwnerOtherKey } from "../../model/beneficial.owner.other.model";
+import { ManagingOfficerCorporate, ManagingOfficerCorporateKey } from "../../model/managing.officer.corporate.model";
+import { ManagingOfficerIndividual, ManagingOfficerKey } from "../../model/managing.officer.model";
+import { startPaymentsSession } from "../../service/payment.service";
+import { getTransaction } from "../../service/transaction.service";
+import { mapTrustApiReturnModelToWebModel } from "../../utils/trusts";
+
+export const getResumePage = async (req: Request, res: Response, next: NextFunction, isFeatureActive: boolean, resumePage: string) => {
```

Comment:
seems strange to pass in isFeatureActive

Reply:
it was passed in to allow the featureFlag status to be passed in